src/components/carousel/index.tsx

```diff
@@ -151,6 +161,16 @@ const Carousel: FC<Props> = (props) => {
 
   const prerenderFallbackSlide = startIndex !== 0 && !mainCarouselRef;
 
+  let carouselHeight = 'full';
```

Comment:
optional: how about:
```typescript
const carouselHeightByPaginationTypeMap = {
   none: "full",
   thumbnail: "calc(100% - 7.5rem)",
   number: 'calc(100% - 5rem)'
}
```

and then access it like: `height={carouselHeightByPaginationTypeMap[paginationType]}`